src/HMRServer.js

```diff
@@ -69,6 +72,15 @@ class HMRServer {
     }
   }
 
+  handleSocketError(err) {
+    if (err.code === 'ECONNRESET') {
+      // This gets triggered on page refresh, ignore this
+      return;
+    }
+    // TODO: Use logger to print errors
+    console.log(prettyError(err));
```

Comment:
have you seen any other errors actually happen? should we just ignore them?

Reply:
This was just for future error handling when adding new features.
Normally ECONNRESET is the only error that accures due to chrome not properly closing the connection

Reply:
Other errors might be parser errors, like an invalid frame, an inflate error if you use permessage-deflate, a frame with a payload bigger than the `maxPayload` option or other `net.Socket` errors like write EPIPE, write ECONNABORTED, etc.